### ISSUE TITLE
request: Disable scenario name in CI variant

### ISSUE THREAD

**Issue by vrnvorona** [Betterment/alchemist#109]
### Is there an existing feature request for this?

- [X] I have searched the existing issues.

### Command

Ability to disable text widget for csenario name in CI env

### Description

Config setting which when enabled removes Text widget with scenario.name parameter from CI image

### Reasoning

Since in CI font is obscured for compativility, names doesn't make sense and just clutter image. Also, ahem font takes much more space and something like 30-35 long name is making CI image twice as wide
<img width="741" alt="image" src="https://github.com/Betterment/alchemist/assets/11422913/440c71af-0543-4c43-b920-00bd0e873253"> 
(this is 35 symbols including whitespaces)

**Comment by TDuffinNTU** [comment]
Agreed. I'd even prefer to have an option to display the text _only_ for section titles, at the risk of minor diff errors.
